src/search/views/Search.tsx

```diff
@@ -15,6 +15,7 @@ import {
 	Navbar,
 	Select,
 	Spacer,
+	Spinner,
```

Comment:
Wordt niet gebruikt

Reply:
already fixed in latest bundle branch: feature/AVO2-374-duplicate-bundle-and-collection